lib/commonBindings.js

```diff
@@ -19,6 +19,8 @@
  *
```

Comment:
CNR entry missing

Reply:
Fixed in 292d293860f9f23dd860cfb60b0111605df0f7f6